python/tank/templatekey.py

```diff
@@ -269,7 +277,8 @@ def __init__(self,
                  shotgun_field_name=None,
                  exclusions=None,
                  abstract=False,
-                 length=None):
+                 length=None,
+                 strict_matching=None):
```

Comment:
Why is the default set to None and not set to `True`? This seems a little confusing.